drivers/adc/ad469x/ad469x.c

```diff
@@ -911,6 +911,59 @@ int32_t ad469x_config(struct ad469x_dev *dev, struct
 	return 0;
 }
 
+/**
+ * @brief Configure analog input high Z mode
+ * @param dev The device structure
+ * @param ch Channel ID
+ * @param status Status of analog input high Z bit
+ * @return 0 in case of success, negative error code otherwise
+ */
+int32_t ad469x_configure_ain_high_z(struct ad469x_dev *dev, uint8_t ch,
+				    enum ad469x_ain_high_z status)
+{
+	uint8_t channel_id;
+
+	if (dev->ch_sequence == AD469x_standard_seq)
+		channel_id = 0;
+	else
+		channel_id = ch;
+
+	return ad469x_spi_write_mask(dev,
+				     AD469x_REG_CONFIG_IN(channel_id),
```

Comment:
code can be rewritten to be more compact, something like:
```
...
AD469x_REG_CONFIG_IN(dev->ch_sequence == AD469x_standard_seq ? 0 : ch),
...
```
see also for get API

Reply:
@ro

> code can be rewritten to be more compact, something like:
> 
> ```
> ...
> AD469x_REG_CONFIG_IN(dev->ch_sequence == AD469x_standard_seq ? 0 : ch),
> ...
> ```
> 
> see also for get API

@rbolboac - There was a discussion in one of the [previous PRs](https://github.com/analogdevicesinc/no-OS/pull/1740#discussion_r1169834815), suggesting to use if..else in place of ternary operators for better readability. What do you suggest?

Reply:
@amiclaus any preference in this case?

Reply:
In this case the operation is simple and straight forward (unlike in the situation https://github.com/analogdevicesinc/no-OS/pull/1740#discussion_r1169834815) therefore ternary operator can be used to simplify things.

Reply:
Modified the implementation to use ternary operator.